extensions/lifx/package.json

```
@@ -0,0 +1,93 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "lifx",
+  "title": "LIFX",
+  "description": "Control your lifx lights with this extension",
+  "icon": "lifx-extension-icon.png",
+  "author": "felixb1010",
+  "categories": [
+    "Fun",
+    "Applications",
+    "Other",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Manage Lights",
+      "subtitle": "LIFX",
+      "description": "Toggle lights on and off",
+      "mode": "view"
+    },
+    {
+      "name": "viewScenes",
+      "title": "Set Scene",
+      "subtitle": "LIFX",
+      "description": "Set the chosen scene",
+      "mode": "view"
+    },
+    {
+      "name": "viewEffects",
+      "title": "Set Effect",
+      "subtitle": "LIFX",
+      "description": "Set the chosen effect",
+      "mode": "view"
+    },
+    {
+      "name": "turnOnAllLights",
+      "title": "Turn On All Lights",
+      "subtitle": "LIFX",
+      "description": "Turn on your lifx lights",
+      "mode": "no-view"
+    },
+    {
+      "name": "turnOffAllLights",
+      "title": "Turn Off All Lights",
+      "subtitle": "LIFX",
+      "description": "Turn off your lifx lights",
+      "mode": "no-view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "lifx_token",
+      "title": "LIFX Token",
+      "description": "Your LIFX token",
+      "type": "textfield",
+      "default": "Token",
```

```suggestion
      "placeholder": "xfergfrtetrektopfksdopfewr",
```
...Or something like that 😃